docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,7 @@ The following table lists all known issues that are currently active and affecti
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- | --------------------- |
+| When you use content bundles to provision a new cluster without using the local Harbor registry, it's possible for the images to be pulled from external networks instead of from the content bundle. If your Edge host has no connection to the internet, some pods may enter the `ImagePullBackOff` state at first, but eventually the pods will be created using images from the content bundle.                                                                                      | No workarounds                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | July 11, 2024  | Edge                  |
```

Comment:
```suggestion
| When you use content bundles to provision a new cluster without using the local Harbor registry, it's possible for the images to be pulled from external networks instead of from the content bundle. If your Edge host has no connection to the internet, some pods may enter the `ImagePullBackOff` state at first, but eventually the pods will be created using images from the content bundle.                                                                                      | No workaround.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | July 11, 2024  | Edge                  |
```

Added a period to "workaround" and removed the s.

Reply:
After reading the issue description i feel like there is a workaround and confirmed it with Venkatesh